cfme/tests/pod/test_appliance_crud.py

```diff
@@ -443,30 +443,6 @@ def test_crud_pod_appliance(temp_pod_appliance, provider, setup_provider):
     assert navigate_to(proj, 'Dashboard')
 
 
-@pytest.mark.ignore_stream('5.10')
-def test_crud_pod_appliance_ansible_deployment(temp_pod_ansible_appliance, provider,
```

Comment:
please don't remove it. I guess they should change ansible template to 5.10 soon.